packages/logos-docusaurus-theme/src/client/components/mdx/JobList/useFetchJobs.ts

```diff
@@ -0,0 +1,108 @@
+import { useState } from 'react'
+
+// Inspired by:
+// https://github.com/acid-info/rnd-dynamic-list/blob/master/react/useFetchJobs.js
+
+const jobBoards = [
+  'acidtest',
+  'logos',
+  'status',
+  'nimbus',
+  'codex',
+  'nomos',
+  'statusnetwork',
+  'ift',
+  'vac',
+  'waku',
+] as const
+
+export type JobBoard = (typeof jobBoards)[number]
+
+const jobBoardMappings: Record<JobBoard, string> = {
```

Comment:
We do not need to hardcode all the names here. Here we would only need to pull the jobs from one specific job baords and that can be specified on a component as is explicitly.